packages/api-gateway/src/drivers/http/adapters/index.js

```diff
@@ -1,9 +1,17 @@
-'use strict';
-
+// Internal dependencies
 const mathAdapters = require('./math/handlers');
+const geolocationAdapters = require('./geolocation/handlers');
 const statusAdapters = require('./status/handlers');
+const adminPanelAdapters = require('./admin_panel/handlers');
+const notificationsAdapters = require('./notifications/handlers');
+const placesAdapters = require('./places/handlers');
+const bookingAdapters = require('./booking/handlers');
```

Comment:
Missing reference to messageAdapters
```suggestion
const bookingAdapters = require('./booking/handlers');
const messageAdapters = require('./messages/handlers');
```